src/cart/crt_init.c

```diff
@@ -1076,19 +1076,9 @@ crt_na_config_init(bool primary, crt_provider_t provider,
 	}
 
 	if (interface) {
-		if (provider == CRT_PROV_OFI_CXI) {
-			D_INFO("Interface '%s' ignored for CXI. Using domain '%s' instead\n",
-			       interface, domain);
-
-			/* Note: crt_provider_iface_str_get() returns interface name  */
```

Comment:
Previously with CXI (as it uses domain names not interface names) we would overwrite interface with domain values, but this causes issues with multi-interface feature that uses interface names as identifiers.